lib/openstax/aws/stack.rb

```diff
@@ -220,6 +222,18 @@ def apply_change_set(params: {}, wait: false)
       change_set
     end
 
+    def revert_to_previous_change_set(wait: false)
+      logger.info("**** DRY RUN ****") if dry_run
+
+      if @previous_parameters
+        logger.info("Applying previous parameters...")
+        apply_change_set(params: @previous_parameters, wait: wait)
+        @previous_parameters = nil
+      else
+        logger.info("There are no saved previous parameters for #{name} stack.")
```

Comment:
Should this be an error? `logger.error`?  And if an error should we exit or raise?

Reply:
I would not call an `error` here and just let the user know that he called a method on a stack which does not have a previous version
If you want me to change that let me know what should happen in that case